tests/unit/StaffNotation.spec.ts

```diff
@@ -114,13 +189,13 @@ describe('StaffNotation', () => {
         disabled: false
       },
       {
-        type: 'administrative-dissolution',
-        label: 'Administrative Dissolution',
+        type: 'put-back-on',
+        label: 'Put Back On',
         disabled: false
       },
       {
-        type: 'put-back-on',
-        label: 'Put Back On',
+        type: 'administrative-dissolution',
+        label: 'Administrative Dissolution',
         disabled: false
```

Comment:
any specific reason these two were swapped? Order matters in anyway?

Reply:
No reason. I originally removed it to make the test case work and then found another solution so I put it back.

Reply:
No biggie but good to avoid unnecessary changes (not adding much value) :)
If possible, restore the way it was

Reply:
Done